config/default/core.entity_view_display.node.localgov_newsroom.default.yml

```diff
@@ -17,6 +17,11 @@ targetEntityType: node
 bundle: localgov_newsroom
 mode: default
 content:
+  content_moderation_control:
```

Comment:
Yes please - it must have been removed at some point in the past on this one, but reintroduced here - we've removed content moderation from other content types in #102 and it's our intention that it shouldn't appear in any of the views. This is an admin task so it can be done in the admin interface.

Reply:
Curiouser and curiouser!

I'm not that bothered, it's fine. I'll create a ticket in the DevOps backlog to look at it, but it's not an immediate priority. I didn't realise it only occurred when the content is archived - that explains why we don't normally see it.

Reply:
Not necessarily when archived, btw. I think only when the content is unpublished - so archived, draft or review. :-)